src/common/routes.tsx

```diff
@@ -93,6 +93,17 @@ export const routes = (
       </Route>
       <Route path="/vendors">
         <Route path="" element={<Vendors.Vendors />} />
+        <Route path="create" element={<Vendors.Create />} />
+
+        <Route path=":id">
+          <Route path="" element={<Vendors.Edit includeTabs />} />
```

Comment:
I suppose this is related to the tab handling and outlet I mentioned in the GitHub comment. I never used anything like `includeTabs` so there's definitely no need for this.